backend/Tasque.Core.Common/Entities/TaskPriority.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Tasque.Core.Common.Entities.Abstract;
+
+namespace Tasque.Core.Common.Entities;
+
+public class TaskPriority : BaseEntity
```

Comment:
Should be enum. We will not change them often. Same for State and Type.

Reply:
Nope. It shouldn't be enum. I was told to made them as tables.